ext/README.md

```diff
@@ -3,6 +3,35 @@
 CEL extensions are a related set of constants, functions, macros, or other
 features which may not be covered by the core CEL spec.
 
+## Encoders
+
+Encoding utilies for marshalling data into standardized representations.
+
+### Base64.Decode
+
+Decodes a base64-encoded string to an unencoded string value.
```

Comment:
I guess I'm showing my age - I first read that as "uuencoded", which seemed like an odd choice.

Base64 decodes to arbitrary binary data, so this should be bytes.  If it's actually ASCII or UTF8 data, the call can be wrapped with `string()`.

Reply:
Fair enough, I'll decode to `bytes`